onedal/__init__.py

```diff
@@ -40,7 +40,7 @@
     _is_dpc_backend = False
 
 
-__all__ = ['decomposition', 'ensemble', 'neighbors', 'primitives', 'svm']
+__all__ = ['decomposition', 'ensemble', 'kmeans', 'neighbors', 'primitives', 'svm']
```

Comment:
KMeans should be in cluster submodule like in stock sklearn
```suggestion
__all__ = ['decomposition', 'ensemble', 'cluster', 'neighbors', 'primitives', 'svm']
```

Reply:
Done